phovea_server/json_decimal_encoder.py

```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Caleydo - Visualization for Molecular Biology - http://caleydo.org
+# Copyright (c) The Caleydo Team. All rights reserved.
+# Licensed under the new BSD license, available at http://caleydo.org/license
+###############################################################################
+
+from builtins import object
+from decimal import Decimal
+
+
+class JSONDecimalEncoder(object):
+  def __contains__(self, obj):
+    if isinstance(obj, Decimal):
```

Comment:
why not just `return isinstance(obj, Decimal)`

Reply:
to avoid unneeded files, I would include it in the other `json_encoder` file but specify another factory method